nengo/spa/tests/test_module.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+import nengo
+from nengo import spa
+from nengo.spa.utils import similarity
+
+
+class SpaCommunicationChannel(spa.Module):
```

Comment:
It's just a class / module used in the test functions... I don't think documentation is necessary.